testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -26,21 +27,164 @@ List<int> _to64(num value) {
+      : assert(window != null),
+        super(window) {
+    constructScenario(window, text, id);
+  }
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+    builder.pushOffset(0, 0);
+    builder.pushClipRect(const Rect.fromLTRB(100, 100, 400, 400));
+    finishBuilderByAddingPlatformViewAndPicture(builder, 1);
+  }
+}
+
+/// Platform view with clip rrect.
+class PlatformViewClipRRectScenario extends PlatformViewScenario {
+  /// Constuct a platform view with clip rrect scenario.
+  PlatformViewClipRRectScenario(Window window, String text, {int id = 0})
+      : super(window, text, id: id);
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+
+    builder.pushOffset(0, 0);
+    builder.pushClipRRect(
+      RRect.fromLTRBAndCorners(
+        100,
+        100,
+        400,
+        400,
+        topLeft: const Radius.circular(15),
+        topRight: const Radius.circular(50),
+        bottomLeft: const Radius.circular(50),
+      ),
+    );
+    finishBuilderByAddingPlatformViewAndPicture(builder, 2);
+  }
+}
+
+/// Platform view with clip path.
+class PlatformViewClipPathScenario extends PlatformViewScenario {
+  /// Constuct a platform view with clip rrect scenario.
+  PlatformViewClipPathScenario(Window window, String text, {int id = 0})
+      : super(window, text, id: id);
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+
+    builder.pushOffset(0, 0);
+
+    // Create a path of rectangle with width of 200 and height of 300, starting from (100, 100).
+    //
+    // The left side of the rectangle becomes a symmetric curve towards the left.
+    // The right side of the rectangle becomes a double curve. From top of bottom of the curve, it goes left then right.
+    //     _______
+    //    |      |
+    //   |      |
+    //  |        |
+    //   |        |
+    //    |______|
+    Path path = Path();
+    path.moveTo(100, 100);
+    path.quadraticBezierTo(50, 250, 100, 400);
+    path.lineTo(350, 400);
+    path.cubicTo(400, 300, 300, 200, 350, 100);
+    path.close();
+    builder.pushClipPath(path);
+
+    finishBuilderByAddingPlatformViewAndPicture(builder, 3);
+  }
+}
+
+/// Platform view with transform.
+class PlatformViewTransformScenario extends PlatformViewScenario {
+  /// Constuct a platform view with transform scenario.
+  PlatformViewTransformScenario(Window window, String text, {int id = 0})
+      : super(window, text, id: id);
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+
+    builder.pushOffset(0, 0);
+    final Matrix4 matrix4 = Matrix4.identity();
+    matrix4.rotateZ(1.0);
+    matrix4.scale(0.5, 0.5, 1.0);
+    matrix4.translate(1000.0, 100.0, 0.0);
+
+    final Float64List matrix4_64 =  Float64List.fromList(matrix4.storage);
```

Comment:
You don't need the copy here - just do `builder.pushTransform(matrix4.storage)`

Reply:
matrix4.storage is a Float32List, `builder.pushTransform(matrix4.storage)` throws a compile error.

Reply:
Ahh - make sure you import `vector_math_64.dart` :)